templates/cf-apache.service.mustache

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=CFEngine Enterprise Webserver
+After=syslog.target
+After=cf-postgres.service
+ConditionPathExists={{{vars.sys.workdir}}}/httpd/bin/apachectl
+PartOf=cfengine3.service
+
+[Service]
+Type=forking
+ExecStart={{{vars.sys.workdir}}}/httpd/bin/apachectl start
+ExecStop={{{vars.sys.workdir}}}/httpd/bin/apachectl stop
+PIDFile={{{vars.sys.workdir}}}/httpd/logs/httpd.pid
+Restart=always
+RestartSec=10
+
+[Install]
+WantedBy=multi-user.target
+WantedBy=cfengine3.service
```

Comment:
The reason for having the templates in masterfiles is so that they can be easily tweaked by users. For example to run cf-serverd in verbose mode you can simply update the template and the unit will be re-rendered and the service re-started.